modules/oc-mirror-enclave-support-about.adoc

```diff
@@ -19,4 +19,35 @@ Enclave support has the following benefits:
 
 * You can mirror content directly from the centralized internal registry to enclaves without restarting the mirroring process from the internet for each enclave.
 
-* You can minimize data transfer between network stages, so to ensure that a blob or image is transferred only once from one stage to another.
\ No newline at end of file
+* You can minimize data transfer between network stages, so to ensure that a blob or image is transferred only once from one stage to another.
+
+== How to mirror to an enclave
+
+image::445_OpenShift_Enclave_support_0724.png[Enclave Support]
+
+The above illustration outlines the flow for using the oc-mirror plugin in different environments, including environments with and without an internet connection.
+
+*Environment with Internet Connection*:
+
+- The user executes oc-mirror plugin V2 to mirror content from an online registry to a local disk directory.
+- The mirrored content is saved to the disk for transfer to offline environments.
```

Comment:
So in this case I think that because this flow implies an order of actions (as in, you must take the first action before you can take the second), ordered/numbered list items would be more appropriate here:
```suggestion
. The user executes oc-mirror plugin V2 to mirror content from an online registry to a local disk directory.
. The mirrored content is saved to the disk for transfer to offline environments.
```

See [this section](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#ordered-lists) of our ISG for more info.